src/main/resources/database/changes/release-10.49.16/add_job_completed_column.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE sample.collectionexercisejob ADD COLUMN jobcomplete boolean;
+UPDATE sample.collectionexercisejob SET jobcomplete = 't';
+ALTER TABLE sample.collectionexercisejob ALTER COLUMN jobcomplete SET NOT NULL;
```

Comment:
Hopefully no one tries to execute a collection exercise just before a deployment